easybuild/easyconfigs/y/yelp-tools/yelp-tools-42.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MesonNinja'
+
+name = 'yelp-tools'
+version = '42.1'
+
+homepage = 'https://gitlab.gnome.org/GNOME/yelp-tools'
+description = """yelp-tools is a collection of scripts and build utilities to help create,
+manage, and publish documentation for Yelp and the web. Most of the heavy
+lifting is done by packages like yelp-xsl and itstool. This package just
+wraps things up in a developer-friendly way."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://gitlab.gnome.org/GNOME/yelp-tools/-/archive/%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['4776766816aaa4fff5a9be7229d03e2444fca2f60a11f645c6171abe8bb73925']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('Meson', '1.1.1'),
+    ('Ninja', '1.11.1'),
+    ('CMake', '3.26.3'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('yelp-xsl', '42.1'),
+    ('ITSTool', '2.0.7'),
+    ('libxml2', '2.11.4'),
+    # docbook-xsl ?
+    ('mallard-ducktype', '1.0.2'),
+    ('lxml', '4.9.2'),
+]
+
+sanity_check_paths = {
+    'files': [
+        'bin/yelp-build',
```

Comment:
@PetrKralCZ Please add a sanity check command for these binaries, like `yelp-build --help`.

I gave that a quick try, and it fails for me:

```
$ yelp-build --help
Traceback (most recent call last):
  File "/software/yelp-tools/42.1-GCCcore-12.3.0/bin/yelp-build", line 29, in <module>
    import lxml.etree
ModuleNotFoundError: No module named 'lxml.etree'
```

Reply:
fixed in 7a1e6e9